config/configuration.json

```diff
@@ -362,7 +362,7 @@
       "tags":"coastline"
     }
   },
-    "GenericTagCheck": {
+  "GenericTagCheck": {
```

Comment:
nit: Looks like your other PR(https://github.com/osmlab/atlas-checks/pull/622) includes this change.